core/src/test/java/com/graphhopper/routing/util/EncodingManagerTest.java

```diff
@@ -66,6 +71,13 @@ public void testEncoderAcceptNoException()
     }
 
     @Test
+    public void testEncoderWithWrongVersionIsRejected()
+    {
+        thrown.expect(IllegalArgumentException.class);
```

Comment:
AFAIK there’s no way to test for the exception message in the annotation, right?